### PR TITLE
Marketplace add & update: Kepler Builder & WireGuard

### PR DESCRIPTION
**Pull request by hmorris3293**
## Description
This PR adds the fakespec & svgs for Kepler Builder & update Wireguard  for Marketplace

## Type of Change
 * Non breaking change ('update', 'change')

### CODE REVIEW

File: packages/manager/src/features/OneClickApps/FakeSpec.ts [hunk 1]
```diff
@@ -1028,13 +1047,13 @@ export const oneClickApps: OCA[] = [
     logo_url: 'assets/webuzo_color.svg'
   },
   {
-    name: 'WireGuard',
-    description: `Configuring WireGuard is as simple as configuring SSH. A connection is established by an
+    name: 'WireGuard®',
```
Comment (abalarin) [inline packages/manager/src/features/OneClickApps/FakeSpec.ts:1050]:
```suggestion
    name: 'WireGuard&reg',
```

File: packages/manager/src/features/OneClickApps/FakeSpec.ts [hunk 2]
```diff
@@ -1028,13 +1047,13 @@ export const oneClickApps: OCA[] = [
...
+    description: `Configuring WireGuard® is as simple as configuring SSH. A connection is established by an
```
Comment (abalarin) [inline packages/manager/src/features/OneClickApps/FakeSpec.ts:1051]:
```suggestion
    description: `Configuring WireGuard&reg is as simple as configuring SSH. A connection is established by an
```

**Review by DevDW — APPROVED** [review]
LGTM

**Review by tiffwong — APPROVED** [review]

**Review by ndukatz — APPROVED** [review]
looks good as far as I can tell. e2e tests pass.

**Comment by Jskobos** [comment]
Made a release branch for this and moving it to staging.

**Pull request closed without merging**